spec/features/edit_etd_spec.rb

```diff
@@ -35,6 +35,7 @@
       fill_in 'Description', with: etd[:description].first
       # term for license URI set in factory
       select('Creative Commons BY-SA Attribution-ShareAlike 4.0 International', from: 'License')
+      fill_in 'Publisher', with: etd[:publisher].first
```

Comment:
@jenlindner is this related to the date work? Or do we need to remove it separately to push this forward?

Reply:
No, we need to remove it separately.